src/aura/RD2_EntryForm/RD2_EntryFormHelper.js

```diff
@@ -10,39 +10,39 @@
             parentId = helper.getParentId();
             component.set('v.parentId', parentId);
         }
-        
-        $A.createComponent("c:rd2EntryForm", {parentId, recordId},
-        function(content, status, error) {
-            if (status === "SUCCESS") {
-                const modalBody = content;
 
-                let modalReference = component.find('overlayLib').showCustomModal({
-                    body: modalBody,
-                    cssClass: component.getName() + ' slds-modal_medium custom-modal',
-                    showCloseButton: true,
-                    closeCallback: function() {
-                        helper.redirectToPage(component);
-                    }
-                });
-                component.set('v.modal', modalReference);
-            } else {
-                console.error(JSON.stringify(error));
-            }
-        });
+        $A.createComponent("c:rd2EntryForm", { parentId, recordId },
+            function (content, status, error) {
+                if (status === "SUCCESS") {
+                    const modalBody = content;
+
+                    let modalReference = component.find('overlayLib').showCustomModal({
+                        body: modalBody,
+                        cssClass: component.getName() + ' slds-modal_medium custom-modal',
+                        showCloseButton: true,
+                        closeCallback: function () {
+                            helper.redirectToPage(component);
+                        }
+                    });
+                    component.set('v.modal', modalReference);
+                } else {
+                    console.error(JSON.stringify(error));
+                }
+            });
     },
 
     /**
     * @description Handle events sent from the modal
     */
-    handleModalEvent: function(component, event) {
+    handleModalEvent: function (component, event) {
```

Comment:
The VS Code formatter automatically updated files on save. Should we all use this formatter for LWC/LCs to avoid code review comments due to the coding style?